src/documents/IoCTestingFramework.md

```diff
@@ -333,25 +577,60 @@ fluid.defaults("fluid.tests.asyncTestTree", {
 });
 ```
 
+### `markupFixture` property supporting fixtures written against markup in the host document
```

Comment:
The sample in your pastie works fine for me with Infusion 2.0.0. Here is the output: 

Infusion at path E:\source\gits\gpii-express\node_modules\infusion is at top level
19:57:00.795:  Registering module gpii-express from path E:\source\gits\gpii-express
19:57:00.797:  Registering module infusion from path E:\source\gits\gpii-express\node_modules\infusion
19:57:00.797:  Registering module infusion from path E:\source\gits\gpii-express\node_modules\infusion
19:57:00.904:  Successfully queued test Simple sequence of 4 active elements
19:57:00.927:  I will be executed first
19:57:00.930:  I come after the beginning
19:57:00.931:  I am the original sequence, in the middle
19:57:00.931:  I am at the end, just before the check
19:57:00.947:  jq: Test concluded - Module "Priority-driven grade budding" Test name "Simple sequence of 4 active elements": 1/1 passed - PASS
19:57:00.949:  jq: ***************
19:57:00.949:  jq: All tests concluded: 1/1 total passed in 75ms - PASS
19:57:00.949:  jq: ***************

Perhaps you could help me investigate.

I used node 6.9.2 x64 on Windows